plugins/metrics/src/main/java/org/apache/cloudstack/response/HostMetricsResponse.java

```diff
@@ -43,6 +43,10 @@
     @Param(description = "the total cpu allocated in Ghz")
     private String cpuAllocated;
 
+    @SerializedName("loadAverage")
+    @Param(description = "the average cpu load the last minute")
+    private String loadAverage;
```

Comment:
Shouldn't this be a Double?

Reply:
I don't think so. I think it's just for output. The other variables here are either Booleans or Strings

Reply:
Ah, yes. I misread. This is the API response to the outside world.